packages/go_router/doc/type-safe-routes.md

```diff
@@ -0,0 +1,73 @@
+Instead of using URL strings to navigate, go_router supports
+type-safe routes using the go_router_builder package. 
+
+To get started, add go_router_builder, build_runner, and build_verify to the
+dev_dependencies section of your pubspec.yaml:
+
+```yaml
+dependency_overrides:
+  go_router_builder: ^1.0.15
```

Comment:
I wonder if there is a way to not hard code this? I was worrying that we may forget to update this  when we publish new version of go_router that will break the go_router_builder 1.0.15.

Reply:
Perhaps just include

Reply:
Let's just use `any` here.